tests/js/lib/request.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var fluid = require("infusion");
+
+var kettle = require("kettle");
+kettle.loadTestingSupport();
+
+fluid.defaults("gpii.express.tests.request", {
```

Comment:
Yes, that's right. If the thing can actually be concretely executed as a test fixture, it goes in "tests". If it can't, it is "test". Naturally this isn't a completely hard and fast rule if there are multiple layers of derivation between the framework and the fixture - but it's generally relatively clear if something is infrastructural (e.g. "kettle.test.request") or an actual test (e.g. "kettle.tests.KETTLE34dataSource")